usaspending_api/references/tests/test_guide.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+
+from django.core.management import call_command
+
+
+@pytest.fixture()
+def guide_data(db):
+    call_command('load_guide')
+
+
+def test_guide_endpoint(client, guide_data):
+
+    # Sorry to squash these together, but I don't want to load the guide data
+    # multiple times
+
+    resp = client.get('/api/v1/references/guide/')
+    assert resp.status_code == 200
+    assert len(resp.data["results"]) > 70
+
+    resp = client.get(
+        'http://localhost:8000/api/v1/references/guide/agency-identifier/')
```

Comment:
Is the localhost reference here an artifact of local testing that should come out before merging?